src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx

```diff
@@ -93,16 +101,16 @@ class EduBenefitsApp extends React.Component {
       <div className="row">
         {devPanel}
         <Element name="topScrollElement"/>
-        <div className="medium-4 columns show-for-medium-up">
-          <Nav
-              data={data}
-              pages={pageState}
-              chapters={chapters}
-              currentUrl={currentLocation.pathname}/>
+        <div className="medium-8 columns">
```

Comment:
Does this need to be in a separate row, or can it be in the same div as NavHeader?